Alexandre documentation

AlexandreProenca opened the pull request.

felipelfb committed:
Create api-documentation.raml

Create api-documentation.raml with data types and query parameters for the endpoints required.

felipelfb committed:
Add description and responses to endpoints

felipelfb committed:
Add description and responses to endpoints

felipelfb committed:
Merge branch 'development-felipeb' of https://github.com/python-aceleradev-squad5/LoanManagement-squad5 into development-felipeb

felipelfb committed:
Add number format and fix datetime type

Add number format for various number type data and fix datetime type data (from date to datetime).

felipelfb committed:
Add GET methods and fix POST methods

felipelfb committed:
Add HTML version of documentation

felipelfb committed:
Update HMTL api documentation with GET methods

AlexandreProenca committed:
First commit

TheCamilovisk commented on api-documentation.raml:
```diff
@@ -0,0 +1,186 @@
+      body:
+        application/json:
+          properties:
+            payment:
```
Field could be a boolean
Name could be made our missed

felipelfb committed:
Minor fixes to documentation

Changed ids to string since we'll use UUID. Changed property payment of object payment to a boolean.  Removed a couple of blank lines.

felipelfb committed:
Add GET method for loans

Added GET method for /loans. Minor fixes all around.

felipelfb committed:
Updated documentation HTML

Updated with changes on the documentation.raml file.
